torch/onnx/symbolic.py

```diff
@@ -497,6 +499,41 @@ def reform_weights(g, w, n, intervals):
     return g.op('Concat', *slices, axis_i=0)
 
 
+def Elman_RNN_symbolic_builder(nonlinearity, input_size, hidden_size, num_layers, batch_first, dropout, bidirectional, **kwargs):
+    if batch_first:
+        return _unimplemented("RNN", "batch_first")
+    if dropout:
+        return _unimplemented("RNN", "dropout")
+    if bidirectional:
+        return _unimplemented("RNN", "bidirectional")
+
+    def symbolic(g, input, all_weights, h0, **fkwargs):
+        # TODO elide this argument to increase parametricity. This is
+        # nontrivial because we provide subsequent optional arguments,
+        # and ONNX does not have a mechanism for skipping non-trailing
+        # optional arguments.
+        sequence_len, batch_size = input.type().sizes()[0:2]
+        sequence_lens = g.op('Constant', value_t=torch.IntTensor(batch_size).fill_(sequence_len))
```

Comment:
btw, we should fix this part of export as well - model should definitely work with different batch sizes (not in this diff of course)

Reply:
yes, agreed